kernel-modules/probe/Makefile

```diff
@@ -0,0 +1,53 @@
+#
+# Copyright (C) 2021 The Falco Authors.
+#
+# This file is dual licensed under either the MIT or GPL 2. See
+# MIT.txt or GPL.txt for full copies of the license.
+#
+
+always-y += probe.o
+# kept for compatibility with kernels < 5.11
+always = $(always-y)
+
+LLC ?= llc
+CLANG ?= clang
+
+KERNELDIR ?= /lib/modules/$(shell uname -r)/build
+
+ifndef FALCO_DIR
+$(error FALCO_DIR must be set)
+endif
+
+# DEBUG = -DBPF_DEBUG
+
+# Begin StackRox
+ifeq ($(shell grep -sq "^\s*struct\s\+audit_task_info\s\+\*audit;\s*$$" $(KERNELDIR)/include/linux/sched.h ; echo $$? ), 0)
+	KBUILD_CPPFLAGS+= -DCOS_73_WORKAROUND
+endif
+# clang-7 does not support -fmacro-prefix-map
+KBUILD_CPPFLAGS:=$(filter-out -fmacro-prefix-map=%,$(KBUILD_CPPFLAGS))
+# End StackRox
```

Comment:
Think we could clean this up a little and make it look the same way as the upstream change? (We should be pulling that soon).

Reply:
Good point - I'd completely forgotten about that change, I'll update it.